### PR TITLE
CTSKF-749 host dev on cloud platform

### PR DESCRIPTION
**Pull request by AlexRyanShep**
#### What

This PR should set up the basics of having an actions workflow, helm chart and dockerfile. Some issues with the dockerfile and asset precompiling remain, but those don't prevent this from working for the purposes of this ticket.

#### Ticket

[CTSKF-749](https://dsdmoj.atlassian.net/browse/CTSKF-749)

[CTSKF-749]: https://dsdmoj.atlassian.net/browse/CTSKF-749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -69,4 +69,4 @@ group :test do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+gem 'tzinfo-data'
```
Comment (jrmhaig) [inline Gemfile:73]: Does this need to be changed? The gem is only required for Windows.

**Review by jrmhaig — APPROVED** [review]
It might be worth squashing some of these commits, if you have the time.

File: Gemfile (hunk 1 above)
Comment (AlexRyanShep) [reply Gemfile:73]: This was causing an error during the Docker build if it was left as it was

**Comment by AlexRyanShep** [comment]
> It might be worth squashing some of these commits, if you have the time.

I've done this, squashed everything down to just major changes

**Review by jrmhaig — APPROVED** [review]

**Pull request merged**